parachains/runtimes/bridge-hubs/bridge-hub-kusama/src/bridge_hub_config.rs

```diff
@@ -0,0 +1,232 @@
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with Cumulus.  If not, see <http://www.gnu.org/licenses/>.
+
+//! Bridge definitions.
+
+use crate::{
+	BridgeParachainPolkadotInstance, Runtime, WithBridgeHubPolkadotMessagesInstance, XcmRouter,
+};
+use bp_messages::LaneId;
+use bridge_runtime_common::{
+	messages,
+	messages::{
+		source::FromBridgedChainMessagesDeliveryProof, target::FromBridgedChainMessagesProof,
+		MessageBridge, ThisChainWithMessages, UnderlyingChainProvider,
+	},
+	messages_xcm_extension::{XcmBlobHauler, XcmBlobHaulerAdapter},
+	refund_relayer_extension::{
+		ActualFeeRefund, RefundBridgedParachainMessages, RefundableMessagesLane,
+		RefundableParachain,
+	},
+};
+use frame_support::{parameter_types, RuntimeDebug};
+use xcm::{latest::prelude::*, prelude::NetworkId};
+use xcm_builder::{BridgeBlobDispatcher, HaulBlobExporter};
+
+parameter_types! {
+	pub const MaxUnrewardedRelayerEntriesAtInboundLane: bp_messages::MessageNonce =
+		bp_bridge_hub_kusama::MAX_UNREWARDED_RELAYERS_IN_CONFIRMATION_TX;
+	pub const MaxUnconfirmedMessagesAtInboundLane: bp_messages::MessageNonce =
+		bp_bridge_hub_kusama::MAX_UNCONFIRMED_MESSAGES_IN_CONFIRMATION_TX;
+	pub const BridgeHubPolkadotChainId: bp_runtime::ChainId = bp_runtime::BRIDGE_HUB_POLKADOT_CHAIN_ID;
+	pub PolkadotGlobalConsensusNetwork: NetworkId = NetworkId::Polkadot;
+	// see the `FEE_BOOST_PER_MESSAGE` constant to get the meaning of this value
+	pub PriorityBoostPerMessage: u64 = 352_085_958_485_958;
+}
+
+/// Proof of messages, coming from BridgeHubPolkadot.
+pub type FromBridgeHubPolkadotMessagesProof =
+	FromBridgedChainMessagesProof<bp_bridge_hub_polkadot::Hash>;
+/// Messages delivery proof for BridgeHubPolkadot messages.
+pub type ToBridgeHubPolkadotMessagesDeliveryProof =
+	FromBridgedChainMessagesDeliveryProof<bp_bridge_hub_polkadot::Hash>;
+
+/// Dispatches received XCM messages from other bridge
+pub type OnThisChainBlobDispatcher<UniversalLocation> =
+	BridgeBlobDispatcher<XcmRouter, UniversalLocation>;
+
+/// Export XCM messages to be relayed to the otherside
+pub type ToBridgeHubPolkadotHaulBlobExporter = HaulBlobExporter<
+	XcmBlobHaulerAdapter<ToBridgeHubPolkadotXcmBlobHauler>,
+	PolkadotGlobalConsensusNetwork,
+	(),
+>;
+pub struct ToBridgeHubPolkadotXcmBlobHauler;
+impl XcmBlobHauler for ToBridgeHubPolkadotXcmBlobHauler {
+	type MessageSender =
+		pallet_bridge_messages::Pallet<Runtime, WithBridgeHubPolkadotMessagesInstance>;
+
+	type MessageSenderOrigin = super::RuntimeOrigin;
+
+	fn message_sender_origin() -> Self::MessageSenderOrigin {
+		pallet_xcm::Origin::from(MultiLocation::here()).into()
+	}
+
+	fn xcm_lane() -> LaneId {
+		// TODO: rework once dynamic lanes are supported (https://github.com/paritytech/parity-bridges-common/issues/1760)
+		STATEMINE_TO_STATEMINT_LANE_ID
```

Comment:
I think it'd be more clear to update to new naming, like `KUSAMA_TO_POLKADOT_ASSET_HUBS_LANE_ID`.

Reply:
yes, thank you, those are leftovers also README.md I am going to change
KUSAMA_TO_POLKADOT_ASSET_HUBS_LANE_ID,
POLKADOT_ASSET_HUBS -> could indicate that there are more hubs on polkadot side,
changed simply to:
ASSET_HUB_KUSAMA_TO_ASSET_HUB_POLKADOT_LANE_ID
ASSET_HUB_POLKADOT_TO_ASSET_HUB_KUSAMA_LANE_ID